lib/convenient_service/rspec/matchers/custom/export.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module ConvenientService
+  module RSpec
+    module Matchers
+      module Custom
+        class Export
+          ##
+          # @param method_name [Symbol, String]
```

Comment:
Please, use `full_name` since names can be complex like `"foo.bar.baz"`.

Thanks.